add-chat-push-notifications/README.md

```diff
@@ -0,0 +1,12 @@
+---
+page_type: sample
+languages:
+- swift
+products:
+- azure
+- azure-communication-services
+---
+
+# Quickstart: Add Chat to your App
+
+For full instructions on how to build this code sample from scratch, please follow `Prerequisites`, `Implementation` and `Testing` section in [Enable Push Notifications in your chat app](https://docs.microsoft.com/en-us/azure/communication-services/tutorials/add-chat-push-notifications).
```

Comment:
You should not link to `en-us` documentation, and instead have it language neutral, for example `https://docs.microsoft.com/azure/communication-services/tutorials/add-chat-push-notifications`

Reply:
agree. Change pushed.